packages/fixie/src/main.ts

```diff
@@ -215,7 +215,7 @@ corpus
       } else if (owner === 'public') {
         ownerType = 'OWNER_PUBLIC';
       }
-      const result = await client.listCorpora(ownerType, opts.offset, opts.limit);
+      const result = await client.listCorpora(ownerType, opts?.offset, opts?.limit);
```

Comment:
This one isn't quite right, you also need to spread ...opts

(see https://github.com/fixie-ai/ai-jsx/pull/466)